app/scripts/lib/environment.js

```diff
@@ -91,6 +91,14 @@
                 );
     },
 
+    /**
+     * Detects if environment is about:accounts
+     * @returns {boolean}
+     */
+    isAboutAccounts: function () {
+      return !! (this.window.top && this.window.name === 'remote');
```

Comment:
The check should be `this.window.top !== this.window`. If the page is not in an iframe, `this.window.top === this.window`